Alignment/TrackerAlignment/plugins/MisalignedTrackerESProducer.cc

```diff
@@ -59,33 +63,31 @@ class MisalignedTrackerESProducer : public edm::ESProducer {
 
 //__________________________________________________________________________________________________
 MisalignedTrackerESProducer::MisalignedTrackerESProducer(const edm::ParameterSet& p)
-    : theSaveToDB(p.getUntrackedParameter<bool>("saveToDbase")),
+    : geomDetToken_(setWhatProduced(this).consumes()),
```

Comment:
@makortel for my own education, is this syntax acceptable also when there are multiple tokens, or shall I instead use a different one:

```c++
auto cc = setWhatProduced(this);
geomDetToken_ = cc.consumes();
ptpToken_ = cc.consumes();
topoToken_ = cc.consumes();
```
in the body of the constructor?

Reply:
The `setWhatProduced(this)` should be called exactly once, so you need to use the other pattern of

> ```c++
> auto cc = setWhatProduced(this);
> geomDetToken_ = cc.consumes();
> ptpToken_ = cc.consumes();
> topoToken_ = cc.consumes();
> ```
> 
> in the body of the constructor

Reply:
thanks, will change accordingly.